bouncer/src/repo/manipulator/modelconvertor/export/repo_model_export_asset.cpp

```diff
@@ -199,7 +224,24 @@ bool AssetModelExport::generateTreeRepresentation(
 		assetListTree.addToTree(MP_LABEL_DATABASE, scene->getDatabaseName());
 		assetListTree.addToTree(MP_LABEL_PROJECT, scene->getProjectName());
 		jsonTrees[assetListFile] = assetListTree;
+
+		std::string errMsg;
+		if (success &= commitUnityAssets(
```

Comment:
It doesn't make sense to commit the unit assets here. So after all I don't think we should be passing the handler here.... (sorry 😛)

This is called during initialisAssetBuffer, which is called by the BouncerWrapper to "setup" all the files and info needed to build the bundle, if for some reason this failed, we don't want to get into a situation where we have already committed a part of the work.

I think what's best to do is return it like the `jsonFiles`/`serialisedFaceBuf`, let BouncerWrapper hold it (like everything else it's holding), and pass it back down to bouncer library when it calls `commitAssetBundleBuffers` in `save AssetBundles`